src/taskgraph/main.py

```diff
@@ -767,10 +767,17 @@ def image_digest(args):
     action="store_true",
     default=False,
     help="Setup the task but pause execution before executing its command. "
-    "Repositories will be cloned, environment variables will be set and an"
+    "Repositories will be cloned, environment variables will be set and an "
     "executable script named `exec-task` will be provided to resume task "
     "execution. Only supported for `run-task` based tasks.",
 )
+@argument(
+    "--develop",
```

Comment:
nit: `--develop` doesn't really describe what this does. I can imagine some users preferring `--develop`, but adding an alias like `--use-local-checkout` might also be nice.